yaml.properties

```diff
@@ -1,12 +1,18 @@
 category=Languages
 description=Allows the analysis of YAML documents
 homepageUrl=https://github.com/sbaudoin/sonar-yaml
-archivedVersions=1.4.3,1.5.0,1.5.1
-publicVersions=1.5.2
+archivedVersions=1.4.3,1.5.0,1.5.1,1.5.2
+publicVersions=1.6.0
 
 defaults.mavenGroupId=com.github.sbaudoin
 defaults.mavenArtifactId=sonar-yaml-plugin
 
+1.6.0.description=Allows rule customization from a local .yamllint file and fixes some documentation inconsistencies
+1.6.0.sqVersions=[7.9,8.9.*,9.0]
```

Comment:
The brackets (`[...]`) format describes a range, so this value isn't valid. Your choices are (note that a point version of 9.0 has been released):
* `[7.9,8.9.*],9.0.*`
* `[7.9,9.0.*]`

I suggest the latter

Reply:
Ok, thanks for the details, I go for the latter as well. New commit to come.